Improve readme

damianooldoni opened the pull request:
This PR aims to improve the README.

damianooldoni committed:
Improve readme

Fix #1

damianooldoni committed:
Add WWW as developer organization

PietrH reviewed (changes requested):
Maybe add some more to the description :smiley:

damianooldoni committed:
Add more info

PietrH reviewed (changes requested):
Please add Peter instead

damianooldoni committed:
Peter instead of Pieter

https://github.com/damianooldoni/git-course-demo/pull/2#pullrequestreview-1787061833
